test/test_core.py

```diff
@@ -5545,7 +5545,7 @@ def test_fcntl(self):
 
   @also_with_nodefs_both
   def test_fcntl_open(self):
-    self.do_run_in_out_file_test('fcntl/test_fcntl_open.c')
+    self.do_run_in_out_file_test('fcntl/test_fcntl_open.c', 'it worked!')
```

Comment:
You don't need the second argument with `do_run_in_out_file_test` since the expected text will some from `fcntl/test_fcntl_open.out`.